### ISSUE TITLE
Zektor, Space Fury, Tac/Scan, and Eliminator show volume control and pause after pressing start

### ISSUE THREAD

**Issue by NightmareSteam** [libretro/mame2003-libretro#310]
I have tried the fix listed for Eliminator in the compatibility list for lr-mame2003 for Retropie but the games still pause and  bring up mames volume control after pressing start. Attempting to change the inputs for pause and config menu causes the games to constantly pause and bring up random parts of the config menu. Disabling samples seems to fix the issue for Tac/Scan and Eliminator but not for Zektor and Space Fury.

**Comment by arcadez** [comment]
Hmmm sound likes like a problem in the core input section of this MAME core, there have been
alot of changes in that dept in the last year or two, personally if it was me that had made said
changes i'd rollback all of them back now and then add each change one at a time to see which 
if any are behind the input breakage of so many games in this core.

**Comment by NightmareSteam** [comment]
After changing the inputs for pause, config, menu, and on screen display the pausing and volume control popping up no longer occurs. I also had to change the inputs for confirm and cancel otherwise whenever I would press the button to bring up the menu it would go straight into the general configuration menu and would not let me exit out of that menu back into the main menu. The games would add credits to the machine when the games reached a point where the digitized voices play like during the Zektor attract mode and that would cause the game to not play through the full attract mode until I changed the inputs for the insert credit buttons. Given that all 4 of the games are Sega G80 based and that this issue does not occur with other games it seems to just be an issue with Sega G80 titles. I tried lr-mame2003 in the windows release of retroarch and it has the same issue.
